src/main/java/blusunrize/immersiveengineering/common/blocks/metal/ConveyorBeltTileEntity.java

```diff
@@ -237,6 +240,12 @@ public boolean isRSPowered()
 		return super.isRSPowered();
 	}
 
+	@Override
+	public BlockPos getModelOffset(BlockState state)
+	{
+		throw new UnsupportedOperationException();
+	}
+
```

Comment:
Why are conveyors implementing this at all?

Reply:
I originally added `getModelOffset` to `IGeneralMultiblock`, which conveyors seem to implement. And when I moved it to a different interface I accidentally added it here instead of removing the implementation.